### PR TITLE
Selecting authorisation method view

### PR DESCRIPTION
**Pull request by berezkin88**

### CODE REVIEW

File: src/main/java/de/adorsys/xs2a/adapter/ui/controller/PageController.java [hunk 1]
```diff
@@ -86,4 +88,22 @@ public String pinInput(@RequestBody String pin) {
         //TODO add appropriate logic
         return "pin";
     }
+
+    @GetMapping("/page/auth-methods")
+    public String authMethod(Model model) {
+
+        // this is a sample data for demonstration purposes only, should be replaced with an appropriate logic in further development
+        Map<String, String> methods = Map.of("SMS-TAN", "901", "chipTAN comfort", "904",
```
Comment (spetrychenko) [inline src/main/java/de/adorsys/xs2a/adapter/ui/controller/PageController.java]: I think key and value should swap places

File: src/main/resources/templates/auth-methods.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+</head>
+<body>
+
+<div class="container pt-5 pb-5 d-md-flex justify-content-center">
+    <form class="form-inline">
+        <div class="input-group">
+            <label for="auth-method-selected" class="mr-3 h5">Please choose one of available methods</label>
+            <select class="custom-select mr-3" id="auth-method-selected">
+                <th:block th:each="method : ${methods}">
+                    <option th:text="${method.getKey()}" th:value="${method.getValue()}"></option>
```
Comment (spetrychenko) [inline src/main/resources/templates/auth-methods.html]: i think key and value should swap places

File: src/main/java/de/adorsys/xs2a/adapter/ui/controller/PageController.java (hunk 1 above)
Comment (berezkin88) [reply src/main/java/de/adorsys/xs2a/adapter/ui/controller/PageController.java]: gotcha, on it...

Comment (berezkin88) [reply src/main/java/de/adorsys/xs2a/adapter/ui/controller/PageController.java]: done

File: src/main/resources/templates/auth-methods.html (hunk 2 above)
Comment (berezkin88) [reply src/main/resources/templates/auth-methods.html]: done

**Pull request merged**